extensions/pom.xml

```diff
@@ -19,7 +19,7 @@
     <description>Mule extensions that augment the core Mule functionality through the Mule Extension API</description>
 
     <modules>
-        <!-- <module>validation</module> -->
+         <module>validation</module>
```

Comment:
Should we start with the right first step by placing this extensions on a different git repo?